mlrun/model_monitoring/db/tsdb/tdengine/tdengine_connector.py

```diff
@@ -186,34 +188,55 @@ def get_model_endpoint_real_time_metrics(
     def get_records(
         self,
         table: str,
-        start: str,
-        end: str,
+        start: datetime,
+        end: datetime,
         columns: typing.Optional[list[str]] = None,
         filter_query: str = "",
+        interval: str = "",
+        agg: typing.Optional[list] = None,
+        limit: int = 0,
+        sliding_window: str = "",
```

Comment:
Whatever parameters are added here should be added in the base class as well. And the types should match too.